echopype/convert/set_groups_ek80.py

```diff
@@ -203,12 +206,25 @@ def set_sonar(self, beam_group_count=1) -> xr.Dataset:
             for param in params:
                 var[param].append(data[param])
 
-        # Create dataset
+        # obtain the correct beam_group and corresponding description from beamgroups_possible
+        for idx, beam in enumerate(beam_group_type):
+            if beam is None:
```

Comment:
What is the scenario for this case? I can't come up with a case where the EK80 key 'descr' does not have keys. Isn't it hardwired into the EK80 `beamgroups_possible` and the way `beam_group_type` is assigned?